sdk/core/src/main/java/com/klaviyo/coresdk/config/KlaviyoConfig.kt

```diff
@@ -28,10 +28,31 @@ class MissingPermission(permission: String) : Exception("You must declare $permi
  * Stores all configuration related to the Klaviyo Android SDK.
  */
 object KlaviyoConfig : Config {
-    private const val DEBOUNCE_INTERVAL: Int = 500
-    private const val NETWORK_TIMEOUT_DEFAULT: Int = 500
-    private const val NETWORK_FLUSH_INTERVAL_DEFAULT: Int = 60000
-    private const val NETWORK_FLUSH_DEPTH_DEFAULT: Int = 20
+    /**
+     * Default value: Debounce time for fluent profile setter methods
+     */
+    private const val DEBOUNCE_INTERVAL: Int = 100
+
+    /**
+     * Default value: Network request timeout duration
+     */
+    private const val NETWORK_TIMEOUT_DEFAULT: Int = 10_000
```

Comment:
Is it shameful that I never knew you could do this underscore style with integers in Kotlin? That's super nice for readability

Reply:
Right?? I had no idea I saw it when I Googled ok https default timeout

Reply:
python too